From 845e79a2a79966609a8009631bce7da71a8c9de7 Mon Sep 17 00:00:00 2001
From: Arun Raghavan <arun@asymptotic.io>
Date: Thu, 9 Jan 2025 15:53:16 -0500
Subject: [PATCH] meson: Install some new headers

The main header has moved to api/audio/... and there are a bunch of new
dependent headers.
---
 webrtc/api/meson.build      | 6 ++++++
 webrtc/rtc_base/meson.build | 5 +++++
 2 files changed, 11 insertions(+)

diff --git a/webrtc/api/meson.build b/webrtc/api/meson.build
index 042ef1b..b6686f1 100644
--- a/webrtc/api/meson.build
+++ b/webrtc/api/meson.build
@@ -18,9 +18,15 @@ api_sources = [
 
 api_headers = [
   ['', 'array_view.h'],
+  ['', 'location.h'],
+  ['', 'ref_count.h'],
   ['', 'scoped_refptr.h'],
+  ['audio', 'audio_processing.h'],
+  ['audio', 'audio_processing_statistics.h'],
   ['audio', 'echo_canceller3_config.h'],
   ['audio', 'echo_control.h'],
+  ['task_queue', 'task_queue_base.h'],
+  ['units', 'time_delta.h'],
 ]
 
 foreach h : api_headers
diff --git a/webrtc/rtc_base/meson.build b/webrtc/rtc_base/meson.build
index 1477af1..bcaf8d1 100644
--- a/webrtc/rtc_base/meson.build
+++ b/webrtc/rtc_base/meson.build
@@ -26,11 +26,16 @@ base_headers = [
   [ '', 'arraysize.h' ],
   [ '', 'checks.h' ],
   [ '', 'ref_count.h' ],
+  [ '', 'thread_annotations.h' ],
   [ '', 'type_traits.h' ],
+  [ 'numerics', 'divide_round.h' ],
   [ 'numerics', 'safe_compare.h' ],
+  [ 'numerics', 'safe_conversions.h' ],
+  [ 'numerics', 'safe_conversions_impl.h' ],
   [ 'system', 'file_wrapper.h' ],
   [ 'system', 'inline.h' ],
   [ 'system', 'rtc_export.h' ],
+  [ 'units', 'unit_base.h' ],
 ]
 
 foreach h : base_headers
-- 
GitLab

